lib/widgets/drawer/mainDrawer.dart

```diff
@@ -93,13 +94,12 @@ class MainDrawer extends StatelessWidget {
           DrawerListTile(
               iconData: FontAwesomeIcons.bullhorn,
               title: 'Events',
-              onTap: () => {}),
+              onTap: () =>
+                  onListTileTap(context, () => navigateTo(EventScreen()))),
           DrawerListTile(
               iconData: FontAwesomeIcons.discord,
               title: 'Discord',
               onTap: () => {}),
-          DrawerListTile(

```

Comment:
not sure what this was, another settings/events tile added by mistake?

Reply:
Oh that seems like a mistake, can without issue be removed.